wfl/calculators/aims.py

```diff
@@ -48,8 +48,6 @@ class Aims(WFLFileIOCalculator, ASE_Aims):
     **kwargs: arguments for ase.calculators.aims.Aims
         See https://wiki.fysik.dtu.dk/ase/_modules/ase/calculators/aims.html.
     """
-    implemented_properties = ["energy", "forces", "stress"]
```

Comment:
All of the properties implemented by the parent class are accessible from this calculator, no need to restrict them here.

Reply:
sounds good